### DISCUSSION TITLE
Tabs / tile groups?

### DISCUSSION THREAD

**Discussion by AndrzejL-eu** [linuxserver/Heimdall#1215] [Ideas]
Hello.

Any chance of adding Tabs / Groups for tiles?

Something on top of the page below search bar? Same buttons as the normal tiles just so when You click on it You get different subset of tiles that belongs to this group?

I have not seen anything like this in options. Did I missed it?

Kindest regards.

Andrzej

**Comment by KodeStar** [comment]
Something along these lines was implemented in v2.6.0 https://github.com/linuxserver/Heimdall/discussions/1292
